packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -440,6 +423,53 @@ export class AggregatedAttestationPool {
     }
     return attestations;
   }
+
+  private onScrapeMetrics(metrics: Metrics): void {
+    const aggregatedAttestationPoolMetrics = metrics.opPool.aggregatedAttestationPool;
```

Comment:
could make it even more concise by doing this
```suggestion
  private onScrapeMetrics({opPool}: Metrics): void {
    const metrics = opPool.aggregatedAttestationPool;
```